### DISCUSSION TITLE
Latest image for Pi Zero-W Does Not Allow Lan Connection

### DISCUSSION THREAD

**Discussion by Bautar** [rern/rAudio#865] [Fixed Issues]
I was running a January build on several of my Pi Zero-W's but that version would no longer update so I loaded "rAudio-1-RPi0-1-2020214..." & am no longer to connect them via ethernet cable (I have a converter for the Pi's micro USB - to RJ45).  This wouldn't be a big issue except my library is huge & the slower WiFi connection is painfully slow for transferring my library to the SD card.

**Comment by rern** [comment]
Please try: [#860](https://github.com/rern/rAudio-1/discussions/860#discussioncomment-2457687)

**Reply by warp4ever** [reply]
Installed this on Zero W, works fine.
Mind you,  I had to connect to ethernet BEFORE starting the PI
